README.md

```diff
@@ -96,8 +88,8 @@ conda env create --file environment.yml
 conda activate catkit2
 ```
 
-Finally install the package using: 
+Finally install the package using:
 ```
 python setup.py develop
```

Comment:
Do you want to update this to `pip install -e .` as well as you're already on this?

Reply:
Changed in [2f361f4](https://github.com/spacetelescope/catkit2/pull/275/commits/2f361f472f8c1d1d879051fb8f214ff6ecde857e).